src/components/Item/AppItem/getIframeSrc.js

```diff
@@ -11,5 +11,12 @@ export const getIframeSrc = (appDetails, item, itemFilters) => {
         iframeSrc += `&userOrgUnit=${ouIds.join(',')}`
     }
 
+    // Add period (pe) to iframeSrc
+    if (itemFilters[FILTER_PE] && itemFilters[FILTER_PE].length) {
+        const peValues = itemFilters[FILTER_PE].map(x => x.id).join(';');
```

Comment:
To match the format of the org unit parameter, let's use a comma as delimeter:

```suggestion
        const peValues = itemFilters[FILTER_PE].map(x => x.id).join(',');
```